app/client/reducers/auth/loginReducer.ts

```diff
@@ -4,13 +4,9 @@ import { LoginActionTypes } from "./types";
 
 interface LoginState {
   isLoggingIn: boolean;
-  //TODO(refactor): what is token actually? seems to only be used in checking it's not null
   token: null | string;
-  //TODO(refactor): is it number or string?
   userId: null | number;
   email: null | string;
-  //TODO(refactor): is it number or string?
-  vendorId: null | number;
```

Comment:
`vendorId` isn't used anywhere in the codebase and is not returned from the backend.